File: app/src/main/java/com/robotnec/budget/core/domain/Account.java
package com.robotnec.budget.core.domain;

import java.io.Serializable;

import lombok.Data;

/**
 * @author zak <zak@swingpulse.com>
 */
@Data
public class Account implements Serializable, Entity {
    private long id;
    private String name;
    private MoneyAmount amount;
}

File: app/src/main/java/com/robotnec/budget/core/domain/Account.kt
package com.robotnec.budget.core.domain

import org.joda.money.Money
import java.io.Serializable

/**
 * @author zak zak@swingpulse.com>
 */
data class Account(override var id: Long,
                   override var name: String,
                   var amount: Money) : Serializable, Entity
